runtime/src/main/java/com/dylibso/chicory/runtime/Machine.java

```diff
@@ -481,9 +481,6 @@ void eval(List<Instruction> code) throws ChicoryException {
                             var val = stack.pop().asByte();
                             var offset = stack.pop().asInt();
                             var end = (size + offset);
-                            if (end > instance.getMemory().getMaximumSize()) {
```

Comment:
Perhaps `getMaximumSize` should be renamed to reflect it's unit. Something like `getMaximumNumberOfPages`